Cannot build from Source - ChakraCore.h not found

<!--
Thank you for reporting an issue.

This issue tracker is for bugs and issues found within Node.js core.
If you require more general support please file an issue on our help
repo. https://github.com/nodejs/help


Please fill in as much of the template below as you're able.

Version: output of `node -v`
Platform: output of `uname -a` (UNIX), or version and 32 or 64-bit (Windows)
Subsystem: if known, please specify affected core module name

If possible, please provide code that demonstrates the problem, keeping it as
simple and free of external dependencies as you are able.
-->

* **Version**: master node-chakracore
* **Platform**: Mac OS 10.13.4
* **Subsystem**:

<!-- Enter your issue details below this comment. -->
Followed the ./configure and make procedure in your readme. After typing make I get

../deps/chakrashim/include/v8.h:67:10: fatal error: 'ChakraCore.h' file not found

Hmm, I just reproduced this error. Looks like what's actually going on is a dependency on chakracore.h has made its way into the node sources, not the chakrashim sources. I'm not sure why this hasn't affected our CI. @kfarnung do you know what might be causing `async_wrap.cc` to take a dependency on `chakracore.h` which fails now, considering the `#include chakracore.h` in `v8.h` has been there for years?

```
In file included from ../src/node_persistent.h:6:0,
                 from ../src/base_object.h:27,
                 from ../src/async_wrap.h:27,
                 from ../src/async_wrap-inl.h:27,
                 from ../src/async_wrap.cc:22:
../deps/chakrashim/include/v8.h:67:24: fatal error: ChakraCore.h: No such file or directory
```

--- a/deps/chakrashim/chakrashim.gyp
+++ b/deps/chakrashim/chakrashim.gyp
@@ -177,6 +177,7 @@
       'actions': [
         {
           'action_name': 'chakra_js2c',
+          'process_outputs_as_sources': 1,
           'inputs': [
             '<@(library_files)'
           ],

--- a/node.gypi
+++ b/node.gypi
@@ -137,6 +137,7 @@
         'deps/chakrashim', # include/v8_platform.h
       ],
       'dependencies': [
+        'deps/chakrashim/chakracore.gyp:chakracore#host',
         'deps/chakrashim/chakrashim.gyp:chakrashim'
       ],
       'conditions': [

